docs/source/technical_tutorials/access/aws_access.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>"we're downloading it here for simplicity and ease of use."</p><p>we might want to rethink the explanation to talk about how downloading can avoid some memory issues, and not all formats are openable over a network connection.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>